nixos/tests/boot.nix

```diff
@@ -4,10 +4,10 @@
 }:
 
 with import ../lib/testing-python.nix { inherit system pkgs; };
-with import ../lib/qemu-flags.nix { inherit pkgs; };
 with pkgs.lib;
 
 let
+  qemu-common = import ../lib/qemu-common.nix { inherit (pkgs) lib pkgs; };
```

Comment:
Maybe we can use callPackage here without inherit?

Reply:
i did what was done in this commit [`b29c2f9` (#127933)](https://github.com/NixOS/nixpkgs/pull/127933/commits/b29c2f97c37f7cb4a1b3411ff9888a49873597d2)